docs/friendly_urls.md

```diff
@@ -0,0 +1,22 @@
+# Friendly URLs
```

Comment:
Is there any other detail that I should add?

Reply:
No, I like this.